backend/Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR app
 
 COPY mvnw .
 COPY .mvn .mvn
+RUN chmod +x mvnw
```

Comment:
Nie wiem dlaczego, ale bez tego mi nie działa. Raczej nikomu to nie popsuje, a mi naprawi.

Reply:
to daj to zamiast `apk update` w 9. - i tak ją miałem usunąć, a powinno się minimalizować ilość layer-ów